vscode-client/snippets/snippets.json

```diff
@@ -0,0 +1,249 @@
+{
+    "shebang": {
+        "description": "[sh]hebang",
+        "prefix": [
+            "sh",
+            "shebang"
+        ],
+        "body": [
+            "#!/usr/bin/env ${1|bash,sh|}"
+        ]
+    },
+    "if": {
+        "description": "[if]",
+        "prefix": "if",
+        "body": [
+            "if ${1:command}; then",
+            "\t$0",
+            "fi"
+        ]
+    },
+    "if-else": {
+        "description": "[if] [e]lse",
+        "prefix": [
+            "ife",
```

Comment:
> I understand that the intension with these shorthand prefixes like ife is to save keystrokes. I'm personally more a fan of self explanatory prefixes that reduces the cognitive load, compared to saving a few keystrokes.

Can I leave just longer but more readable prefixes like `if-else` instead of smth like `ife`?

Reply:
> Can I leave just longer but more readable prefixes like if-else instead of smth like ife?

The long prefixes is what I advocate for here, so let us keep those. It also matches how a language like TypeScript provides exposes snippets in vscode.

Reply:
Done. :)